pyproject.toml

```diff
@@ -1,9 +1,51 @@
+[project]
+name = "cgsubmit"
+version = "1.2.0"
```

Comment:
you must increase at least the patch version or the releasing will fail. 1.3.0 is also fine